CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 FEATURES:
 
 * `vcd_vapp_vm` - Ability to add metadata to a VM. For previous behaviour please see `BACKWARDS INCOMPATIBILITIES` ([#158](https://github.com/terraform-providers/terraform-provider-vcd/issues/158))
+* `vcd_vapp_vm` - Ability to enable hardware assisted CPU virtualization for VM. It allows hypervisor nesting. [#219]
```

Comment:
Shouldn't we use "GH-" format which is then parsed by the build bot?
https://www.terraform.io/docs/extend/best-practices/versioning.html

Reply:
Sorry. Stepped on my own proposal. Of course I will change it and fix the above comment as well.